Header Header
string nodeName
string logType
string message